packages/dev/tools/README.md

```diff
@@ -1,6 +1,9 @@
 # @blockly/dev-tools [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
 
 A library of common utilities for Blockly extension development.
+- `toolboxSimple`: XML for an always-open flyout with no categories.
+- `toolboxCategories`: XML for a toolbox with multiple categories and all of the default blocks.
+- `BlocklyDebugRenderer`: A visualizer for debugging custom renderers.
```

Comment:
Obvious question: what's our preferred naming scheme here?

Reply:
I think we drop Blockly as it's being imported from ``@blockly/dev-tools``. 
UpperCamelCase for class names (as per style guide). 
so, how does ``DebugRenderer`` sound?

Reply:
sg, will change